libraries/AP_GPS/AP_GPS_ERB.cpp

```diff
@@ -159,8 +159,8 @@ AP_GPS_ERB::_parse_gps(void)
         state.location.alt    = (int32_t)(_buffer.pos.altitude_msl * 100);
         state.status          = next_fix;
         _new_position = true;
-        state.horizontal_accuracy = _buffer.pos.horizontal_accuracy * 1.0e-3f;
-        state.vertical_accuracy = _buffer.pos.vertical_accuracy * 1.0e-3f;
+        state.horizontal_accuracy = _buffer.pos.horizontal_accuracy * 1.0e-3f * 0.83f; // multiply by 0.83 to convert RMS value into CEP value
+        state.vertical_accuracy = _buffer.pos.vertical_accuracy * 1.0e-3f * 0.83f;     // multiply by 0.83 to convert RMS value into CEP value
```

Comment:
Yes, that is part of the problem. Every manufacturer has a slightly different way of calculating this. But it seams that CEP is the most common one. AFAIK uBlox uses CEP as well. The RTKLIB source code calculates RMS values. So in order to make these things comparable convert RMS to CEP.